app/src/main/java/com/example/coctailapp/ui/screens/registerScreen.kt

```diff
@@ -289,7 +190,6 @@ fun registerScreen(navController: NavHostController) {
                     )
                     Text(
```

Comment:
Same as on login screen. Use HorizontalDividers with WEIGHT params instead of Spacers